src/table/cells/handle-key-press.js

```diff
@@ -54,17 +54,47 @@ export const preventDefaultBehavior = (evt) => {
   evt.preventDefault();
 };
 
-const handleKeyPress = (evt, rootElement, rowIndex, colIndex, cell, selState, selDispatch) => {
+export const moveFocus = (evt, rootElement, rowIndex, colIndex, selState) => {
+  preventDefaultBehavior(evt);
+  removeCurrentFocus(evt);
+  const rowAndColumnCount = getRowAndColumnCount(rootElement);
+  const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex, selState);
+  moveToNextFocus(rowAndColumnCount.rowElements, nextRow, nextCol);
+};
+
+export const headHandleKeyPress = (evt, rootElement, rowIndex, colIndex, changeSortOrder, layout, isDim) => {
   switch (evt.key) {
     case 'ArrowUp':
     case 'ArrowDown':
     case 'ArrowRight':
     case 'ArrowLeft': {
+      moveFocus(evt, rootElement, rowIndex, colIndex);
+      break;
+    }
+    // Space bar: update the sorting
+    case ' ': {
       preventDefaultBehavior(evt);
-      removeCurrentFocus(evt);
-      const rowAndColumnCount = getRowAndColumnCount(rootElement);
-      const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex, selState);
-      moveToNextFocus(rowAndColumnCount.rowElements, nextRow, nextCol);
+      changeSortOrder(layout, isDim, colIndex);
+      break;
+    }
+    // Enter: update the sorting
+    case 'Enter': {
+      preventDefaultBehavior(evt);
+      changeSortOrder(layout, isDim, colIndex);
+      break;
```

Comment:
This is identical to space. Do as with the arrows and group them

Reply:
Right. Fixed. Thanks